samples/bluetooth/hci_uart/prj.conf

```diff
@@ -8,8 +8,9 @@ CONFIG_BT=y
 CONFIG_BT_HCI_RAW=y
 CONFIG_BT_HCI_RAW_H4=y
 CONFIG_BT_HCI_RAW_H4_ENABLE=y
-CONFIG_BT_RX_BUF_LEN=258
-CONFIG_BT_DISCARDABLE_BUF_SIZE=257
+CONFIG_BT_BUF_ACL_RX_SIZE=255
```

Comment:
True, this could be set to 251 here, since it is controller only. hci_raw would not be able to use the full buffer since it just forwards LL PDUs as received from the controller.

In theory though HCI_RAW could reassembly LL_PDUs up to this size before sending them to the host, who would further reassemble them into L2CAP PDUs.